InteractiveHtmlBom/i18n/en_UK/language_en.bat

```diff
@@ -0,0 +1,13 @@
+::start up echo
+set thx4using=Thankyou For Using Generate Interactive Bom
```

Comment:
Add a `i18n_` prefix to all of these variables so that it's clear that it's a translation string.
Also don't change the strings from what we already agreed on.